package.json

```diff
@@ -46,6 +46,7 @@
 	"dependencies": {
 		"@11ty/eleventy-plugin-inclusive-language": "^1.0.0",
 		"mappy-breakpoints": "^0.2.3",
+		"node-sass": "^5.0.0",
```

Comment:
Can we revert this and changes to `package-lock.json`? Sass compilation is currently handed by `gulp-sass`.

Additionally, did you run into any issues getting the site to compile because of this? I want to make sure its easy to set up.

Reply:
Yes for sure! I actually was going to give a comment about this: I ran into an issue of not having node-sass package for some reasons, hence I couldn't compile and run the site the first time. I have to clear the node cache and manually add the node-sass package itself for the site to run properly. Do you have any insights on why this might happen and how I should address it?

Reply:
i just deleted the line in package.json and looks like the website compiled fine!! Looks like it was just the first time it compiled that it happened 😅  still curious to know why though!

Reply:
That's super curious. Mentioning https://github.com/a11yproject/a11yproject.com/pull/1141 so we can reference it.